content/id/docs/concepts/workloads/controllers/cron-jobs.md

```diff
@@ -0,0 +1,58 @@
+---
+title: CronJob
+content_template: templates/concept
+weight: 80
+---
+
+{{% capture overview %}}
+
+Suatu _CronJob_ menciptakan [_Job_](/docs/concepts/workloads/controllers/jobs-run-to-completion/) yang dijadwalkan berdasarkan waktu tertentu.
```

Comment:
CronJob and Job are Kubernetes objects. Should be written as camel case without italic (unless it's referring to cron job / job as english word -- not as the kube object).

ref: https://kubernetes.io/docs/contribute/style/style-guide/#use-camel-case-for-api-objects

Same comment applies for the rest on this page